news/2018-03-26-osu-coffee-hour-taiko-gamemode-history.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "osu! Coffee Hour: osu!taiko Gamemode History"
+date: 2018-03-27 05:00:00 +0000
+---
+
+If there's one thing that sets osu! apart from other communities, it's the amount of different stories to be told. This episode of the osu! Coffee Hour will be the second in our Gamemode History series, where we dive into the unique story behind each gamemode. We're dedicating our second Gamemode History to osu!taiko, the second gamemode to join the osu! ranks nearly 10 years ago.
+
+[![](/wiki/shared/news/banners/coffee-hour.jpg)](http://itsalmo.st/#osutaikogamemodeshowcase)
+
+## This week's theme
+
+What better time for osu!taiko than the day before the Taiko World Cup live drawings? Joining me ([HappyStick](https://osu.ppy.sh/users/256802)) this Friday will be [Raiden](https://osu.ppy.sh/users/2239480) and [karterfreak](https://osu.ppy.sh/users/1031958).
+
+Raiden is an osu!taiko mapper, modder and player, active in the mapping and modding scene since June 2015, former Beatmap Nominator and current member of the Quality Assurance Team for osu!taiko. He has participated in the renewal of the old osu!taiko Ranking Criteria, has hosted his own Mentorship Program and Mapping Contest for osu!taiko and participated actively in the translation of the osu!wiki to Spanish. He has also played for Spain in two TWCs, and will play his 3rd on this year.
```

Comment:
Spanish wiki translation is irrelevant here - might want to substitute that for something involving the taiko subcommunity